largest_sum_contiguous_subarray/LargestSumContiguousSubarray.c

```diff
@@ -0,0 +1,26 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int LargestSumContinousSubArray(int a[], int size) {
+	int max_till = 0;
+	int max_int = 0;
+	for (int i = 0; i < size; i++) {
+		max_int = max_int +a[i];
+		if(max_till < max_int) {
+			max_till = max_int;
+		}
+		if (max_int < 0) {
+			max_int = 0;
+		}
+	}
+	return max_till;
+}
+
+int main() {
+	int array [10];
+	for (int k = 0; k < 10; k++)
+	{
```

Comment:
Opening brace should be on the same line as "for loop" declaration (as per your style).